static/css/mozsearch.css

```diff
@@ -262,64 +259,28 @@ body {
      .source-line-with-number div necessary for sticky operation. */
   position: relative;
```

Comment:
It seems like maybe we can remove this `position: relative` and the comment that suggest it's necessary to create a containing block.  Per bullet 1 of https://developer.mozilla.org/en-US/docs/Web/CSS/Containing_Block#Identifying_the_containing_block it seems like the fact that this is `display: block` is sufficient?  (Turning off the `position: relative` certainly doesn't break anything on the current searchfox.org.)

Reply:
Yes, you're right! In fact `display: block` is not even needed, as it's the default for `<div>`s.